DC-322-L-1 Web Engineering Lab/main.py

```diff
@@ -0,0 +1,174 @@
+import utils
+
+
+def registerAccount():
+    """
+        Allows a user to create a new account.
+
+        This function prompts the user to enter a 3-digit account number and a 4-digit password.
+        It checks if the entered account number already exists. If it does, a message is displayed.
+        If the account number is unique, the function creates a new account with an initial balance of 100
+        and stores the account details in a data file.
+
+        Returns:
+        None
+        """
+
+    accountDetails = []
+    utils.registerHeader()
```

Comment:
There should be a level of abstraction.
Use 
from utils import * 
from utils import -some specific function/class etc.

Just call it by name. No need to mention utils. everywhere.